### ISSUE TITLE
DR | Adjust scheduling for status updater jobs

### ISSUE THREAD

**Issue by dfong-adh** [department-of-veterans-affairs/va.gov-team#99356]
## Value Statement

**_As a_** developer

**_I want to_** make sure we are not sending excessive requests to LH endpoints

**_So that_** we minimize the possibility of load spikes and job failures

---

## Background Context
Currently every non-finalized form and evidence submission is polled for status changes every hour. This frequency may not be necessary, and can result in load spikes due to how the jobs are scheduled. Emails for failures are also only sent once a day.

## Tasks

- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Dependencies 

- Add here

## Acceptance Criteria

- [ ] _AC_
- [ ] _AC_
- [ ] _AC_

## Out of scope

- Add here

## Open questions

- Add here

## Designs and Build Notes

- Add here

## Outcome, Success Measure, KPI(S), and Tracking Link

- Add here

### Enablement team (if needed)
@ member of team

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by comaurice** [comment]
Waiting for platform review, per Derek.

**Comment by kayline** [comment]
Made a requested change, waiting on re-review from platform
